composer.json

```diff
@@ -15,12 +15,14 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "firebase/php-jwt": "^6.3"
+        "firebase/php-jwt": "~6.3.0"
```

Comment:
Could we move this proposed change into a separate PR, please?  It's highly debatable and will require some discussion that will block the other expected and fine changes of this PR.

After reverting, ensure to run `composer update --lock`

Reply:
my latest two commits should do the trick!